src/wallet/wallet.cpp

```diff
@@ -483,6 +483,8 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, CWalletDB* pwalletdb)
         CWalletTx& wtx = (*ret.first).second;
```

Comment:
I am ok if you want to update the notation for this to modern standards... ret.first->second while you are in here.

Reply:
Done, I could make a PR with a scripted-diff later to update all usages across the codebase if you want.